code/modules/modular_computers/file_system/programs/generic/ntnrc_client.dm

```diff
@@ -156,6 +166,28 @@
 			channel.password = ""
 		else
 			channel.password = newpassword
+	if(href_list["PRG_directmessage"])
+		. = TRUE
+		var/clients = list()
+		var/names = list()
+		for(var/cl in ntnet_global.chat_clients)
+			var/datum/computer_file/program/chatclient/C = cl
```

Comment:
You should not be able to direct message yourself. There should be an error message if you're the only one active.